Parsers/ASimDns/ProductParsers/ASimDns.yaml

```diff
@@ -15,12 +15,17 @@ References:
 Description: |
   ASIM Source Agnostic DNS Parser
 ParserName: ASimDns
+ParserParams:
+  - Name: disabled
+    Type: bool
+    Default: false
 ParserQuery: |
+  let DisabledParser=toscalar(_GetWatchlist('ASimDisabledParser') | project Parser=column_ifexists('Parser','') | summarize make_set(Parser));
   union isfuzzy=true
     vimDnsEmpty
-    , ASimDnsCiscoUmbrella
-    , ASimDnsInfobloxNIOS
-    , ASimDnsMicrosoftOMS
-    , ASimDnsGcp
-    , ASimDnsCorelightZeek
-    , ASimDnsMicrosoftSysmon
\ No newline at end of file
+    , ASimDnsCiscoUmbrella     (disabled or (DisabledParser has 'CiscoUmbrella'))
```

Comment:
The parser is DnsCiscoUmbrella and so on. CiscoUmbrella can have other parsers.

Reply:
makes sense